style/style.js

```diff
@@ -9,8 +18,8 @@ export const otsTheme = createMuiTheme({
 
 export const mediaquery = {
   mobile: 'max-width: 550px',
-  smallToTablet: 'min-width: 760px',
-  tabletToDesktop: 'min-width: 990px',
-  desktopToBigScreen: 'min-width: 1200px',
-  bigScreenBreakpoint: '1200px',
+  smallToTablet: `min-width: ${customBreakpoints.sm}px`,
+  tabletToDesktop: `min-width: ${customBreakpoints.md}px`,
+  desktopToBigScreen: `min-width: ${customBreakpoints.lg}px`,
+  bigScreenBreakpoint: `${customBreakpoints.lg}px`,
```

Comment:
we can actually remove this 😅 we never used and i'm not really sure what is the idea behind it since we already have `desktopToBigScreen`

Reply:
Yeah, doesn't make much sense...